src/main/java/driver/FtpClient.java

```diff
@@ -0,0 +1,159 @@
+        Properties config = new Properties();
+        try {
+            String configFilePath = "src/main/java/config/local.properties";
+            FileInputStream propsInput = new FileInputStream(configFilePath);
+            config.load(propsInput);
+        } catch (IOException e) {
+            System.out.println("Failed to load config file");
+        }
+        return config.getProperty("REMOTE_PATH");
+    }
+
+    // Establish connection to local FTP server
+    public static FTPClient connectToServer(){
+        // Load local config file
+        Properties config = new Properties();
+        try {
+            String configFilePath = "src/main/java/config/local.properties";
+            FileInputStream propsInput = new FileInputStream(configFilePath);
+            config.load(propsInput);
+        } catch (IOException e) {
+            System.out.println("Failed to load config file");
+        }
+
+        String server = config.getProperty("FTP_HOST");
+        int port = Integer.parseInt(config.getProperty("FTP_PORT"));
+        String user = config.getProperty("FTP_USER");
+        String pass = config.getProperty("FTP_PASS");
+
+        FTPClient ftpClient = new FTPClient();
+
+        try {
+            ftpClient.connect(server, port);
+            showServerReply(ftpClient);
+            int replyCode = ftpClient.getReplyCode();
+            if (!FTPReply.isPositiveCompletion(replyCode)) {
+                System.out.println("Operation failed. Server reply code: " + replyCode);
+                return ftpClient;
+            }
+            boolean success = ftpClient.login(user, pass);
+            showServerReply(ftpClient);
+            if (!success) {
+                System.out.println("Could not login to the server");
+                return ftpClient;
+            } else {
+                System.out.println("LOGGED IN SERVER");
+                return ftpClient;
+            }
+        } catch (IOException ex) {
+            System.out.println("Oops! Something wrong happened");
+            ex.printStackTrace();
+        }
+        return ftpClient;
+    }
+
+    // Upload a file to local FTP server
+    public static boolean uploadFile(String filePath, String fileName){
+        // Establish connection to FTP server
+        FTPClient ftpClient = connectToServer();
+        
+        // Upload file
+        try {
+            ftpClient.enterLocalPassiveMode();
+            ftpClient.setFileType(FTP.BINARY_FILE_TYPE);
+ 
+            File localFile = new File(filePath);           
+            String remoteFile = getRemotePath() + fileName + ".pdf";
+            InputStream inputStream = new FileInputStream(localFile);
+ 
+            System.out.println("Start uploading file");
+            boolean done = ftpClient.storeFile(remoteFile, inputStream);
+            inputStream.close();
+            if (done) {
+                System.out.println("The file is uploaded successfully.");
```

Comment:
I don't think print to the console is necessary here since we are not doing so for other operations in the software. Consistent logging across the project is a feature that we should consider for the future, but this is ok for now

Reply:
Do I remove the print statement or is it ok?